src/main/java/net/earthcomputer/clientcommands/command/AliasCommand.java

```
@@ -0,0 +1,174 @@
+
+public class AliasCommand {
+
+    private static final Logger LOGGER = LogManager.getLogger();
+
+    private static final Path aliasPath = FabricLoader.getInstance().getConfigDir().resolve("clientcommands").resolve("alias_list.json");
+
+    public static void register(CommandDispatcher<ServerCommandSource> dispatcher) {
+        addClientSideCommand("calias");
+
+        dispatcher.register(literal("calias")
+            .then(literal("add")
+                .then(argument("key", string())
+                    .then(argument("command", greedyString())
+                        .executes(ctx -> addAlias(getString(ctx, "key"), getString(ctx, "command"))))))
+            .then(literal("list")
+                .executes(ctx -> listAliases()))
+            .then(literal("remove")
+                .then(argument("key", string())
+                    .executes(ctx -> removeAlias(getString(ctx, "key"))))));
+
+        HashMap<String, String> aliasMap = getAliases();
+
+        if (aliasMap == null || aliasMap.isEmpty()) {
+            sendError(new LiteralText("No aliases registered"));
+        } else {
+            for(String key: aliasMap.keySet()) {
+                addClientSideCommand(key);
+                dispatcher.register(literal(key)
+                        .executes(ctx -> executeAliasCommand(key, null))
+                        .then(argument("arguments", greedyString())
+                                .executes(ctx -> executeAliasCommand(key, getString(ctx, "arguments")))));
+            }
+        }
+
+    }
+    private static int executeAliasCommand(String aliasKey, String arguments) {
+
+        // TODO: add support for optional greedy text arguments(?)
+        HashMap<String, String> aliasMap = getAliases();
+        String cmd;
+        if(aliasMap.containsKey(aliasKey)) {
+            cmd = aliasMap.get(aliasKey);
+        } else {
+            sendError(new LiteralText("No such key exists"));
```

You should probably throw an exception rather than sending an error message.